src/main/java/uk/gov/hmcts/reform/cwrdapi/util/RequestUtils.java

```diff
@@ -30,4 +41,35 @@ public static void trimIdamRoles(List<CaseWorkersProfileCreationRequest> request
             }
         });
     }
+
+    public static PageRequest validateAndBuildPaginationObject(Integer pageNumber,
+                                                               Integer pageSize,
+                                                               String sortColumn,
+                                                               String sortDirection,
+                                                               String loggingComponentName,
+                                                               int configPageSize,
+                                                               String configSortColumn) {
+
+        if (pageNumber != null && pageNumber < 0) {
+            log.info("{}:: Invalid Page Number", loggingComponentName);
+            throw new InvalidRequestException(String.format(INVALID_FIELD, PAGE_NUMBER));
+        }
+        if (pageSize != null && pageSize <= 0) {
```

Comment:
it is trivial but lets be consistent  and use nonNull rather !=null

Reply:
sure